src/main/scala/org/broadinstitute/dsde/workbench/sam/openam/PostgresAccessPolicyDAO.scala

```diff
@@ -362,21 +364,19 @@ class PostgresAccessPolicyDAO(protected val dbRef: DbReference,
     }
   }
 
-  private def insertPolicyActions(actions: Set[ResourceAction], policyId: PolicyPK)(implicit session: DBSession): Int = {
+  private def insertPolicyActions(actions: Set[FullyQualifiedResourceAction], policyId: PolicyPK, descends: Boolean)(implicit session: DBSession): Int = {
```

Comment:
Yup, by default the `.apply()` returns the number of rows updated, so we just return that number instead of dropping it. I'm not sure if we ever really use it to be honest, but it's good practice to return it anyways I think